src/views/AccountDetailNew/index.vue

```diff
@@ -447,7 +447,11 @@ export default {
           if (res === undefined || typeof res !== "object") {
             return Promise.reject();
           }
-          this.tokenDetails = res.balances;
+          this.tokenDetails = res.balances.filter(token => {
+            const {free, assetId} = token;
+            return Number(free) > 0 || (assetId === 1 || assetId === 16000);
```

Comment:
was thinking, if we could query staking asset and use that.. instead of (assetId === 1 || assetId === 16000)..
But it is used as this twice in this file.. so I think its fine

Reply:
yeah, fair point, I'm cutting corners here a bit, plus trying to minimise the code changes.